newrelic.js

```diff
@@ -1,24 +0,0 @@
-/**
- * New Relic agent configuration.
- *
- * See lib/config.defaults.js in the agent distribution for a more complete
- * description of configuration variables and their potential values.
- */
-exports.config = {
-  /**
-   * Array of application names.
-   */
-  app_name : ['ds-mdata-responder-heroku'],
-  /**
-   * Your New Relic license key.
-   */
-  license_key : process.env.NEW_RELIC_LICENSE_KEY,
-  logging : {
-    /**
-     * Level at which to log. 'trace' is most useful to New Relic when diagnosing
-     * issues with the agent, 'info' and higher will impose the least overhead on
-     * production applications.
-     */
-    level : 'trace'
```

Comment:
Refs https://github.com/DoSomething/ds-mdata-responder/issues/518#issuecomment-234054471

It seems `trace` is the source of the [noisy messages](https://github.com/DoSomething/ds-mdata-responder/issues/518#issuecomment-233358812).

When re-installing and setting to `info`, per the instructions, things are looking a lot quieter locally, at least after the server starts. That might be installation issue on my local. 

I see activity in our Newrelic heroku add-on UI, corresponding with the URL i was testing locally:

<img width="1012" alt="screen shot 2016-07-21 at 2 04 41 pm" src="https://cloud.githubusercontent.com/assets/1236811/17038775/2859da52-4f4c-11e6-90d6-a46f3f6d7053.png">

````
1:58:22 PM web.1 |  > node server
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.252Z","msg":"Could not load /Users/aschachter/Development/ds-mdata-responder/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.274Z","msg":"Could not load /Users/aschachter/Development/ds-mdata-responder/node_modules/count-von-count/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.278Z","msg":"Could not load /Users/aschachter/Development/ds-mdata-responder/node_modules/escodegen/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.280Z","msg":"Could not load /Users/aschachter/Development/ds-mdata-responder/node_modules/istanbul/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.283Z","msg":"Could not load /Users/aschachter/Development/ds-mdata-responder/node_modules/newrelic/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.295Z","msg":"Could not load /usr/local/lib/node_modules/firebase-tools/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.319Z","msg":"Could not load /usr/local/lib/node_modules/firebase-tools/node_modules/google-auth-library/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.327Z","msg":"Could not load /usr/local/lib/node_modules/firebase-tools/node_modules/update-notifier/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.328Z","msg":"Could not load /usr/local/lib/node_modules/npm/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.348Z","msg":"Could not load /usr/local/lib/node_modules/npm/node_modules/request/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.352Z","msg":"Could not load /usr/local/lib/node_modules/npm/node_modules/request/node_modules/http-signature/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.356Z","msg":"Could not load /usr/local/lib/node_modules/react-native-cli/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.359Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.396Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/anchor/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.397Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/anchor/node_modules/geojsonhint/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.400Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/anchor/node_modules/geojsonhint/node_modules/jsonlint-lines/node_modules/nomnom/node_modules/chalk/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.402Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/captains-log/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.405Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/cross-spawn-async/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.412Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/grunt/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.414Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/grunt-contrib-clean/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.414Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/grunt-contrib-coffee/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.416Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/grunt-contrib-concat/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.417Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/grunt-contrib-cssmin/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.417Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/grunt-contrib-jst/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.421Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/grunt-contrib-uglify/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.425Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/maxmin/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.426Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/messageformat/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.427Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/nomnom/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.427Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/noptify/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.430Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/sails-generate/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.432Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/sails-hook-sockets/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.434Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/sails-util/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.435Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/skipper/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.436Z","msg":"Could not load /usr/local/lib/node_modules/sails/node_modules/skipper-disk/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":40,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.441Z","msg":"Could not load /Users/aschachter/.local/share/heroku/cli/lib/node_modules/.bin/package.json","component":"environment"}
1:58:22 PM web.1 |  {"v":0,"level":30,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:22.505Z","msg":"Starting New Relic for Node.js connection process."}
1:58:22 PM web.1 |  { [Error: Cannot find module '../build/Release/bson'] code: 'MODULE_NOT_FOUND' }
1:58:22 PM web.1 |  js-bson: Failed to load c++ bson extension, using pure JS version
1:58:23 PM web.1 |  Mongoose connected to connectionOperations with status code: 1
1:58:23 PM web.1 |  Mongoose connected to connectionConfig with status code: 1
1:58:23 PM web.1 |  info: Express server listening on port 5000 in thor mode...
1:58:24 PM web.1 |  info: Successfully logged in to Phoenix API. 
1:58:24 PM web.1 |  	sessid: [removed]
1:58:24 PM web.1 |  	session_name: [removed] 
1:58:24 PM web.1 |  	token: [removed]
1:58:24 PM web.1 |  {"v":0,"level":30,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:24.944Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/1212848/applications/12429178","component":"new_relic_response"}
1:58:24 PM web.1 |  {"v":0,"level":30,"name":"newrelic","hostname":"Aarons-MacBook-Pro-2.local","pid":2476,"time":"2016-07-21T20:58:24.944Z","msg":"Connected to collector-205.newrelic.com:443 with agent run ID 86143499020901578.","component":"collector_api"}
````